t/35-script_clone_job.t

```diff
@@ -155,8 +155,7 @@ subtest 'asset download' => sub {
         qr|downloading|s, 'download not attempted if openQA reports missing assets';
     }
     qr|assets are missing.*iso/foo.iso|s, 'error about missing asset shown';
-    unlike $@, qr|hdd/some.qcow2|, 'no error about qcow2 image which is generated by parent job';
```

Comment:
You shouldn't just delete this test

Reply:
I was waiting to see the test results.  but I was about to ask to review this. So if you say no I will revert it. but then I have another problem which I am not sure how to approach. Any ideas?

Reply:
would it make sense to negate the throws_ok?